siege_utilities/reporting/analytics/polling_analyzer.py

```
@@ -108,261 +106,130 @@ def create_longitudinal_analysis(self,
                 dimension_rankings = dimension_rankings.sort_values('sum', ascending=False).head(top_n)
-                
+
                 # Convert to list of tuples
                 rankings[dimension] = [
-                    (row[dimension], row['sum'], row['percentage'], row['growth'])
+                    (row[dimension], row['sum'], row['percentage'])
                     for _, row in dimension_rankings.iterrows()
                 ]
-                
+
             return rankings
```

Ruff (0.15.10)

[warning] Consider moving this statement to an `else` block

(TRY300)